api4/graphql.go

```diff
@@ -101,6 +102,9 @@ func (api *API) graphQL(c *Context, w http.ResponseWriter, r *http.Request) {
 	rolesLoader := dataloader.NewBatchedLoader(graphQLRolesLoader, dataloader.WithBatchCapacity(200))
 	reqCtx = context.WithValue(reqCtx, rolesLoaderCtx, rolesLoader)
 
+	channelsLoader := dataloader.NewBatchedLoader(graphQLChannelsLoader, dataloader.WithBatchCapacity(200))
```

Comment:
Wondering if we should make the capacity a constant, would it make any sense?

Reply:
Yeah .. why not